test/record/associations/hasAndBelongsToManyTest.js

```diff
@@ -88,5 +88,55 @@ describe('Viking.Record::associations', () => {
             });
         });
     });
+    
+    describe('addBang', () => {
+        it('sends request', function () {
+            let model = new Model({id: 24})
+            let parent = new Parent({id: 11})
+            model.association('parents').addBang(parent)
```

Comment:
can just be `model.association`

Reply:
Not sure I follow. Needs to know which association to build the request, yeah?